docs/installation/home-assistant.mdx

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 5
+---
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Screenshot from "../../src/components/Screenshot";
+
+# Home Assistant Addon
+
+:::note
+Diese Anleitung richtet sich an Benutzer, die Home Assistant verwenden und evcc als Addon verwenden möchten. Es ist selbstverständlich auch möglich,
+evcc außerhalb von Home Assistant zu betreiben und auch in Home Assistant zu integrieren. Aber da das Addon sehr wenig Ressourcen benötigt, ist der Betrieb von evcc als
+Addon der einfachste Weg.
+:::
```

Comment:
Auch hier, das ist eher ne Einleitung und muss nicht in eine Infobox.

> evcc außerhalb von Home Assistant zu betreiben und auch in Home Assistant zu integrieren. Aber da das Addon sehr wenig Ressourcen benötigt, ist der Betrieb von evcc als Addon der einfachste Weg.

Das würde ich so nicht schreiben. Der Betrieb von evcc im Home Assistant Addon wird, Stand heute, nicht offiziell von uns unterstützt. Es ist eine Community Integration. Grund dafür ist, dass wir im Fehlerfall an wichtige Daten nicht gut rankommen (fehlende evcc cli). Daher würde ich das hier erst einmal konservativer formulieren. Auch dass es sicher um eine Community Integration handelt sollte klarer werden. Dafür wäre ne kurze Infobox sicher angebracht 😄